charts/vela-core/templates/defwithtemplate/autoscale.yaml

```diff
@@ -18,32 +18,24 @@ spec:
   extension:
     template: |
       output: {
-          apiVersion: "standard.oam.dev/v1alpha1"
-          kind:       "Autoscaler"
-          spec: {
-              minReplicas: parameter.minReplicas
-              maxReplicas: parameter.maxReplicas
-              triggers: [{
-                  name: parameter.name
-                  type: parameter.type
-                  condition: {
-                      startAt:  parameter.startAt
-                      duration: parameter.duration
-                      days:     parameter.days
-                      replicas: parameter.replicas
-                      timezone: parameter.timezone
-                  }
-              }, ...]
-          }
-        }
+      	apiVersion: "standard.oam.dev/v1alpha1"
+      	kind:       "Autoscaler"
+      	spec: {
+      		minReplicas: parameter.minReplicas
+      		maxReplicas: parameter.maxReplicas
+      		triggers: [{
+      			type: "cpu"
```

Comment:
Do we have other resource that have the same format? such as memory ?

Reply:
It could work, but I haven't tried on memory. https://github.com/kubernetes/kubernetes/issues/77575